sys-apps/likwid/likwid-4.3.1.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+FORTRAN_NEEDED=fortran
+
+inherit fcaps linux-info toolchain-funcs fortran-2
+
+DESCRIPTION="A performance-oriented tool suite for x86 multicore environments"
+HOMEPAGE="https://github.com/rrze-likwid/likwid"
+SRC_URI="https://ftp.fau.de/pub/likwid/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="fortran"
+
+RDEPEND="dev-lang/perl"
+
+DEPEND="${RDEPEND}
+	fortran? ( sys-devel/gcc:*[fortran] )
+	dev-lang/lua:0"
+
+CONFIG_CHECK="~X86_MSR"
+
+FILECAPS=(
+	-M 755 cap_sys_rawio usr/sbin/likwid-accessD --
+	-M 755 cap_sys_rawio usr/bin/likwid-{perfctr,bench,powermeter}
+)
+
+# See Bug 558402
+PATCHES=(
+	"${FILESDIR}/${P}-Makefile.patch"
+	"${FILESDIR}/${P}-fix-gnustack.patch"
+	"${FILESDIR}/${P}-config.mk.patch"
+)
+
+src_prepare() {
+	# Set PREFIX path to include sandbox path
+	sed -e 's:^PREFIX = .*:PREFIX = '${D}'/usr:' -i config.mk || \
+		die "Failed to set correct prefix path"
+
+	# Set the path to library directory.
+	sed -e 's:$(get_libdir):'$(get_libdir)':' -i config.mk || \
```

Comment:
Please merge the two `sed`s on the same file. Also no need for `\` when the line ends with `||`.

Reply:
Sure, let me give that a try.